.gitignore

```diff
@@ -14,3 +14,4 @@ replay_pid*
 /.slcache/
 /tomcat.8081/
 /rest-e2e-tests/node_modules/
+web-app/webpack-bundles
```

Comment:
No such directory exists anymore, and no need to add it to `.gitignore`. If it exists on your machine for some reason, just delete it.